python/py-boto3/Portfile

```diff
@@ -20,11 +20,11 @@ homepage            https://github.com/boto/boto3
 master_sites        pypi:b/boto3
 distname            boto3-${version}
 
-checksums           rmd160 4ea15b11359b7fadb16deeac07e336df33d0da39 \
-                    sha256 0a0c0f0859a2be56b23823f8c1d50abf3c89d7d4d054019f24de69eeee9ad75c \
-                    size   93147
+checksums           rmd160  ef020c97f6cfab2fafca691e56065ddb9a227920 \
+                    sha256  3fe790bf548c5a6a816779cf38eb844b7126c24d0c56be6561dab177f3bb5fba \
+                    size    98352
 
-python.versions     27 35 36 37 38
```

Comment:
there is no reason to remove PY version that are not EOL

Reply:
Thanks. Put back.